spec/models/form_spec.rb

```diff
@@ -190,8 +190,6 @@ def answer_local_authority(lettings_log)
         FormHandler.instance.use_real_forms!
 
         example.run
-
-        FormHandler.instance.use_fake_forms!
```

Comment:
I think we might need to set it to avoid flaky tests in other files (?)

Reply:
Since the FormHandler is reinitialized after every test runs from the spec_helper, I'm sceptical this can be doing anything.
I have run all tests several times on local and it's passed -once- twice on remote